src/CSET/operators/plot.py

```diff
@@ -139,9 +150,42 @@ def _plot_and_save_contour_plot(
     # Setup plot details, size, resolution, etc.
     fig = plt.figure(figsize=(15, 15), facecolor="w", edgecolor="k")
 
+    try:
+        with open(colorbar_file, "r+t", encoding="UTF-8") as fp:
+            fcntl.flock(fp, fcntl.LOCK_EX)
+            fp.seek(0)
```

Comment:
```suggestion
        with open(colorbar_file, "rt", encoding="UTF-8") as fp:
```
Do we need to take an exclusive write lock on the colour bar file? We are just reading it right?